app/static/pages/volunteer/work_description_templates/development_description.html

```diff
@@ -1,10 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Any reason you need multiple "DOCTYPE html" in this and other files?

Also -- any reason you need the tag at all? We didn't have it before and I thought it worked fine; we don't use it in the oppia app either.

Reply:
I think I added them because I thought I was going to fail the HTML linter. Turns out I was wrong after all. Thanks Sean :)